pkg/sources/syslog/syslog.go

```diff
@@ -123,7 +123,7 @@ func (s *Source) InjectConnection(conn *sourcespb.Syslog) {
 }
 
 // Init returns an initialized Syslog source.
-func (s *Source) Init(_ context.Context, name string, jobId, sourceId int64, verify bool, connection *anypb.Any, concurrency int) error {
+func (s *Source) Init(_ context.Context, name string, jobId sources.JobID, sourceId sources.SourceID, verify bool, connection *anypb.Any, concurrency int) error {
```

Comment:
nit: any reason sometimes it's `jobID` like in line 52 and sometimes it's `jobId`?

Reply:
No, just ambiguity in what "camel case" means. I asked Ahrav at one point and the official Google style guide recommends `jobID`, so I tend to use that.

https://google.github.io/styleguide/go/decisions#initialisms

I'll make a followup PR to update the variable names, since this PR is just updating the types.